concourse/enumerator.py

```diff
@@ -285,6 +287,8 @@ def _scan_repository_for_definitions(
                     path='.ci/pipeline_definitions',
                     ref=branch_name
                 )
+                # it's necessary to prefix 'heads/' as per github3.py doc
```

Comment:
of course we should use canonical ref-names - no need to document this imo..

Reply:
It's in contrast to the usage two lines above. But that's a github.py detail, you're right. Removed.